warp-element/src/utils.js

```diff
@@ -1,27 +1,32 @@
 import { unsafeCSS } from 'lit';
 
-export const isServer = () => {
+const isServer = () => {
     return !(typeof window !== 'undefined');
 };
 
-export const parseBrand = (str = '') => {
+const parseBrand = (str = '') => {
     if (str.toLocaleLowerCase().includes('blocket')) return { sld: 'blocket', tld: 'se' };
     if (str.toLocaleLowerCase().includes('tori')) return { sld: 'tori', tld: 'fi' };
     if (str.toLocaleLowerCase().includes('finn')) return { sld: 'finn', tld: 'no' };
     if (str.toLocaleLowerCase().includes('dba')) return { sld: 'dba', tld: 'dk' };
     return { sld: 'finn', tld: 'no' };
 };
 
+
+/*
+ * @param {string} [brandStr] - a string with brand information
+ * @returns {import("./global.js").Brand} brand object
```

Comment:
So this works? You can reference the .Brand property because there is a @typedef for Brand in global.js then?

Reply:
As far as I've understood JSDoc, that is how it works according to [jsdoc docs](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html)

Reply:
yea, might well be by the looks of it though slightly hard to tell from those docs but if its working its working so yay.